software/plugins/external-item-search/docs/readme.md

```
@@ -1,2 +1,18 @@
 # Docs
 
+
+## Python Virtual env
+
+```
+$ python3 -m venv ./.venv
+$ source .venv/bin/activate
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)